spec/models/application_draft_spec.rb

```diff
@@ -314,7 +314,8 @@ def assign_project(project)
     end
 
     context 'when the application_draft is valid' do
-      let(:application_draft) { create(:application_draft, :appliable) }
+      let(:team)          { create(:team) }
```

Comment:
Hey @cesswairimu 👋 Gave this a quick look and it seems the issue is in this team you are creating. It needs to be an `:applying_team` for it to work well with the application draft.

But maybe instead of making the team work with `create(:team, :applying_team)` you could use the team that is created for you in the `application_draft`. Our factory already creates a team, so you can access it in your tests and use `applying_team.team.students.first`, for example. This way you could remove the team you created.